docs/en/stack/ml/anomaly-detection/ml-ad-job-types.asciidoc

```diff
@@ -130,3 +131,14 @@ URI path that is visited by very few client IPs in the population (this is the
 reason why it's rare). The client IPs that have many interactions with this URI 
 path are anomalous compared to the rest of the population that rarely interact 
 with the URI path.
+
+
+[discrete]
+[[geo-jobs]]
+== Geo jobs
+
+Geo {anomaly-jobs} detect unusual occurrences in the geographic locations of 
+your data. Your data set must contain geopoint data to be able to use the 
```

Comment:
`geo_point` and `geo_shape` field types are supported. but perhaps that could be summarised by just saying `geo`?

Reply:
Fixed in [c14da70](https://github.com/elastic/stack-docs/pull/2328/commits/c14da708237f5c403dfb1153f32e231fd5151d4b).